src/components/templates/Playa/Playa.tsx

```diff
@@ -1043,7 +1044,7 @@ const Playa = () => {
           className={`playa-videochat ${inVideoChat ? "show" : ""}`}
           style={{ height: videoChatHeight }}
         >
-          <VideoChatLayer setSelectedUserProfile={setSelectedUserProfile} />
+          <VideoChatLayer setSelectedUserProfile={setUserProfile} />
```

Comment:
Technically we should probably just remove this prop entirely + use the hook within this component. but `Playa` is a legacy template that is in the process of being deleted, and `VideoChatLayer` seems to only be called by it, so no need to worry about that here.